mono/cil/cil-opcodes.xml

```diff
@@ -319,4 +319,5 @@
 <opcode name="mono_calli_extra_arg" input="VarPop" output="VarPush" args="InlineSig" o1="0xF0" o2="0x18" flow="call" />
 <opcode name="mono_lddomain" input="Pop0" output="PushI" args="InlineNone" o1="0xF0" o2="0x19" flow="next" />
 <opcode name="mono_atomic_store_i4" input="PopI+PopI" output="Push0" args="InlineI" o1="0xF0" o2="0x1A" flow="next" />
+<opcode name="mono_get_last_error" input="Pop0" output="PushI" args="InlineI" o1="0xF0" o2="0x1B" flow="next" />
```

Comment:
Win32 last error is not in the TLS but in the TEB (thread environment block) at a different offset 0x068, compared to the TLS data slots that are located on offset 0x1480 or 0x1780 depending on TLS key.